exercices/part3/practice1.py

```diff
@@ -0,0 +1,12 @@
+
+
+if __name__ == "__main__":
+    """Write a program that counts for the user. Let the user enter the starting number, the ending number, 
+    and the amount by which to count."""
+
+    start = int(input("Starting number:  "))
+    end = int(input("Ending number: "))
+    count = int(input("Amount by which to count: "))
+
```

Comment:
What happens if user enters a value different that int?

Reply:
an error will be displayed (ValueError: invalid literal for int() with base 10:), because the int recognise only numbers, not letters or symbols